cpu/efm32/Makefile.features

```diff
@@ -1,10 +1,11 @@
 FEATURES_PROVIDED += periph_cpuid
 FEATURES_PROVIDED += periph_flashpage
-FEATURES_PROVIDED += periph_pm
 
 FEATURES_CONFLICT += periph_rtc:periph_rtt
 FEATURES_CONFLICT_MSG += "On the EFM32, the RTC and RTT map to the same hardware peripheral."
 
 ifeq (1,$(EFM32_TNRG))
   FEATURES_PROVIDED += periph_hwrng
 endif
+
+-include $(RIOTCPU)/cortexm_common/Makefile.features
```

Comment:
Do this without minus (don't ignore missing). I know the other platforms have the minus as well, but it is better to do it with error checking so that we will get feedback on typos etc.

Reply:
Done.